bin/openshift-operations

```diff
@@ -7,10 +7,9 @@ require "topological_inventory/openshift/operations/worker"
 queue_host = ENV["QUEUE_HOST"] || "localhost"
 queue_port = ENV["QUEUE_PORT"] || 9092
 
-uri = URI.parse(ENV["TOPOLOGICAL_INVENTORY_URL"])
 TopologicalInventoryApiClient.configure do |config|
-  config.scheme = uri.scheme || "http"
-  config.host = "#{uri.host}:#{uri.port}"
+  config.scheme = "http"
```

Comment:
Should we pass in scheme here as well?  We can default to `'http'` but I could see us eventually moving to SSL for intra-platform service calls.

Reply:
I don't think we'll ever terminate ssl on the service itself. If we do intra-service ssl it will likely be through the service mesh and will be transparent to the app.

Reply:
Plus there's no built-in for scheme so we would have to pass that in through the deployment. We get the host and port for free by just defining the service.